### DISCUSSION TITLE
Table with "table-pin-rows" and/or "table-pin-cols" overlaps drawer-side

### DISCUSSION THREAD

**Discussion by jimmynguyen1308** [saadeghi/daisyui#2157] [Q&A] [answered]
Hi everyone!

I am currently using DaisyUI to create a table. With the feature to pin first row and/or column, I found the library very useful. However, I came across a problem which those pinned rows/columns jump in front of the sidebar `class="drawer-side"` (use case: in mobile view, for example, you have to toggle drawer back & forth).

Does anyone know how to fix this and/or workaround?

Much appreciated!

**Comment by saadeghi** [comment, marked as answer]
Can you please provide a live example on [Tailwind Play](https://daisyui.com/tailwindplay/) or a code example?

**Reply by jimmynguyen1308** [reply]
Hi @saadeghi,

Thanks for the quick reply!
I've just made the live example on Tailwind Play [here](https://play.tailwindcss.com/411MWGhngU).

Once you click the `Open Drawer` button you can see that the pinned heading row of the table is in front of the toggled drawer.

Cheers!

**Reply by saadeghi** [reply]
@jimmynguyen1308 Thank you.  
This can be fixed by adding a z-index (like `z-[2]`) to `drawer-side`.  

It's not a good idea if I add a z-index by default because depending on the content, people need different orders for their elements (for example they may need a sidebar to go on top of a navbar or under a navbar. so it's better if developers choose the z-index as they need.)

**Reply by jimmynguyen1308** [reply]
Hi @saadeghi,

Sorry for the late response.
Thanks for the tip you gave me, it fixed the issue.
However, I think should still be in front of the pinned table rows/cols since I couldn't think of any use cases otherwise.

Anyways, cheers man! Thanks for the support <3
